### PR TITLE
add api lifecycle tags to admissionregistration.k8s.io

### PR DESCRIPTION
**Pull request by deads2k**
Adds the release lifecycle tags to this group.  Be aware, per the existing comment, this API was deprecated in 1.16.  In spite of the comment, I have set the removed date to 1.22 instead of 1.19.

/kind cleanup
@kubernetes/sig-api-machinery-pr-reviews 
/assign @liggitt @lavalamp 

```release-note
NONE
```

### CODE REVIEW

File: build/root/Makefile.generated_files [hunk 1]
```diff
@@ -131,7 +131,7 @@ ifeq ($(DBG_MAKEFILE),1)
     $(warning ***** finding all +k8s:prerelease-lifecycle-gen tags)
 endif
 PRERELEASE_LIFECYCLE_DIRS := $(shell                                             \
-    grep --color=never -l '+k8s:prerelease-lifecycle-gen=' $(ALL_K8S_TAG_FILES)  \
+    grep --color=never -l '+k8s:prerelease-lifecycle-gen' $(ALL_K8S_TAG_FILES)  \
```
Comment (liggitt) [inline build/root/Makefile.generated_files:134]: what happens if there are prerelease-lifecycle-gen tags but not prerelease-lifecycle-gen=true? is prerelease-lifecycle-gen=true even required now?

File: staging/src/k8s.io/api/admissionregistration/v1beta1/types.go [hunk 2]
```diff
@@ -113,6 +113,9 @@ const (
 // +genclient
 // +genclient:nonNamespaced
 // +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+// +k8s:prerelease-lifecycle-gen:introduced=1.9
+// +k8s:prerelease-lifecycle-gen:deprecated=1.16
+// +k8s:prerelease-lifecycle-gen:removed=1.22
```
Comment (liggitt) [inline staging/src/k8s.io/api/admissionregistration/v1beta1/types.go:119]: add `// +k8s:prerelease-lifecycle-gen:replacement=<group>,<version>,<kind>` tags where applicable

File: build/root/Makefile.generated_files (hunk 1 above)
Comment (deads2k) [reply build/root/Makefile.generated_files:134]: > what happens if there are prerelease-lifecycle-gen tags but not prerelease-lifecycle-gen=true? is prerelease-lifecycle-gen=true even required now?

Yeah.  The generator looks for it.  This is a superset selection of all possibles. However, it still didn't do what I want.

**Pull request merged**